nilearn/interfaces/bids/glm.py

```diff
@@ -143,7 +143,7 @@ def save_glm_to_bids(
         design_matrix.to_csv(
             dm_file,
             sep='\t',
-            line_terminator='\n',
+            lineterminator='\n',
```

Comment:
I would be tempted to remove it altogether and run the tests.

It is the only occurrence in the entire code base and I am unclear as to why it would be needed specifically here. But maybe I am missing something.

Reply:
I think we are good removing it since it is set to the default value